core/src/main/scala/com/nvidia/spark/rapids/tool/analysis/AppSQLPlanAnalyzer.scala

```diff
@@ -74,6 +81,18 @@ class AppSQLPlanAnalyzer(app: AppBase, appIndex: Int) extends AppAnalysisBase(ap
     stageToDiagnosticMetrics(stageId)(accum.accMetaRef.getName()) = accum
   }
 
+  /**
+   * Given an input IO diagnostic metric result, update IODiagnosticMetrics mapping
+   * @param accum SQLAccumProfileResults to be analyzed
+   */
+  private def updateIODiagnosticMetrics(accum: SQLAccumProfileResults): Unit = {
```

Comment:
nit: This method seems to be `appending` instead of `updating` the map. Can we rename it to something more specific, such as `addMetricToIoDiagnostics`?

Reply:
thanks, I renamed to `updateIODiagnosticMetricsMap` to be consistent with `updateStageDiagnosticMetrics`, I think adding is a form of update as well. If you think naming with `add` is better, I can change this.

Reply:
Thanks @cindyyuanjiang. That makes sense.